apps/src/aichat/views/chatWorkspace.module.scss

```
@@ -114,6 +114,6 @@ $tabs-default-spacing: 4px;
 }
 
 .modeDropdown {
-  margin: 0 0 10px 10px;
+  margin-left: 10px;
```

For RTL support you can use: 
```suggestion
  margin-inline-start: 10px;
```

---

done!